modules/admin_manual/pages/configuration/general_topics/search.adoc

```diff
@@ -1,59 +1,280 @@
 = Full Text Search 
 :toc: right
+:toclevels: 4
 :page_aliases: configuration/search/index.adoc
 :elastic-search-url: https://www.elastic.co/elasticsearch/
+:elastic-search-install-url: https://www.elastic.co/guide/en/elastic-stack/7.17/index.html
 :search_elastic-app-url: {oc-marketplace-url}/apps/search_elastic 
 :simple-query-string-query-url: https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-simple-query-string-query.html
+:ingest-url: https://www.elastic.co/guide/en/elasticsearch/plugins/current/ingest-attachment.html
+:create-api-key-url: https://www.elastic.co/guide/en/elasticsearch/reference/current/security-api-create-api-key.html#security-api-create-api-key
 
-:description: This page describes how to setup a full text search functionality using a Search Elastic Server. 
+:description: ownCloud offers the ability to use full text search via the Full Text Search app and a Search Elastic Server. This allows users to search not only for file names but also for content within files stored in ownCloud.
```

Comment:
"an Elasticsearch Server"?

Reply:
fixed, thanks for catching 👍

Reply:
Not really, its still `to a Search Elastic Server`

Reply:
fixed, thanks